GSA/made-in-america-data
# Refactor json script

Refactored the `createMappedData` function from a long list of if statements to to a key: value mapping instead. 

To test if function still works:
- comment out the `runner.runscript()` command on line `307`
-  Run tests with `npm run test
- -All test should pass
